### PR TITLE
Added API handlers for events

### PR DESCRIPTION
**Pull request by kevhuang**

### CODE REVIEW

File: client/public/js/utils/EventAPI.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    }, function() {
+      // Return empty array on error
+      return [];
+    });
+  },
+
+  /**
+   * Create a new event
+   * @param {object} event An object containing all the event attributes
+   */
```
Comment (ekong2) [inline client/public/js/utils/EventAPI.js:64]: don't you need to have the event be passed into the ajax data: parameter?

File: client/public/js/utils/EventAPI.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    }, function() {
+      // Return empty array on error
+      return [];
+    });
+  },
+
+  /**
+   * Edit an event
+   * @param {number} eventId The primary key of the event in the database
+   */
```
Comment (ekong2) [inline client/public/js/utils/EventAPI.js:84]: same here. somehow the information that we will update has to be passed into the ajax call right?

File: client/public/js/utils/EventAPI.js (hunk 1 above)
Comment (kevhuang) [inline client/public/js/utils/EventAPI.js:64]: thanks for catching that...i'll refactor

Comment (kevhuang) [inline client/public/js/utils/EventAPI.js:64]: @ekong2 done

**Pull request merged**